### PR TITLE
fixing the kotlin dsl examples on the api components

### PR DESCRIPTION
**Pull request by carlossteinzup**
It fix the kotlin DSL examples on the API components.

### CODE REVIEW

File: content/en/api/components/layout/listview.md [hunk 1]
```diff
@@ -159,48 +159,62 @@ Default value is ListDirection.VERTICAL
...
+                Text("age: @{item.age}"),
+            )
+        ).applyStyle(
+            Style(
+                margin = EdgeValue(bottom = 20.unitReal())
+            )
         )
     )
-)
+}()
```
Comment (vitorbassozup) [inline content/en/api/components/layout/listview.md:217]:
```suggestion
}
```
It looks like this wasn't supposed to be here?

File: content/en/api/components/forms/simple-form.md [hunk 2]
```diff
@@ -89,24 +89,24 @@ Check out the example below on how its implemented a `SimpleForm`
 
 ```javascript
```
Comment (vitorbassozup) [inline content/en/api/components/forms/simple-form.md:90]:
```suggestion
```kotlin
```

File: content/en/api/accessibility.md [hunk 3]
```diff
@@ -47,15 +47,15 @@ If backend does not specify accessibility, the default values ​​for each pla
 
 {{% tab name="Kotlin DSL" %}}
 ```
```
Comment (vitorbassozup) [inline content/en/api/accessibility.md:49]:
```suggestion
```kotlin
```

File: content/en/api/components/ui/button.md [hunk 4]
```diff
@@ -92,7 +92,7 @@ Check the example below to see how the structure works:
 {{% tab name="Kotlin DSL" %}}
 ```
```
Comment (vitorbassozup) [inline content/en/api/components/ui/button.md:93]:
```suggestion
```kotlin
```

File: content/pt/api/components/ui/image/imagepath.md [hunk 5]
```diff
@@ -40,9 +40,11 @@ Irá referenciar uma imagem que já está nos assets locais.
 
 {{% tab name="Kotlin DSL" %}}
 ```javascript
```
Comment (vitorbassozup) [inline content/pt/api/components/ui/image/imagepath.md:42]:
```suggestion
```kotlin
```

**Review by luismaximozup — APPROVED** [review]

**Pull request merged**